fbgemm_gpu/src/merge_pooled_embeddings_gpu.cpp

```diff
@@ -20,6 +20,14 @@
 #include "fbgemm_gpu/sparse_ops_utils.h"
 #include "fbgemm_gpu/topology_utils.h"
 
+// For some reason, hipify fails to replace the macro names when compiling for
+// ROCm, so we manually replace it here.  Name mapping based on:
+// https://github.com/pytorch/pytorch/blob/master/torch/utils/hipify/cuda_to_hip_mappings.py
+#ifdef __HIP_PLATFORM_HCC__
+#define C10_CUDA_CLEAR_ERROR C10_HIP_CLEAR_ERROR
+#define C10_CUDA_ERROR_HANDLED C10_HIP_ERROR_HANDLED
+#endif
```

Comment:
@liligwu It'd be nice if you could upgrade the `hipify_torch` submodule version to remove this change (after this PR is merged).

Reply:
Hi @shintaro-iwasaki, we're evaluating the mapping of these macros internally, I'll reach you back if I got replied. Thank you.

Reply:
Thanks a lot, @liligwu!

Reply:
@q10 @shintaro-iwasaki , please see the PR https://github.com/pytorch/FBGEMM/pull/1682